matador/plotting/convergence_plotting.py

```diff
@@ -11,6 +11,8 @@
 from collections import defaultdict
 from traceback import print_exc
 import numpy as np
+import matplotlib
+matplotlib.use('TkAgg')
```

Comment:
Did you mean to include the changes in this file?

Reply:
no